src/executor/explode_executor.py

```diff
@@ -0,0 +1,65 @@
+# coding=utf-8
+# Copyright 2018-2020 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Iterator
+
+from src.models.storage.batch import Batch
+from src.executor.abstract_executor import AbstractExecutor
+from src.planner.explode_plan import ExplodePlan
+from src.utils.logging_manager import LoggingManager, LoggingLevel
+import pandas as pd
+import numpy as np
+
+class ExplodeExecutor(AbstractExecutor):
+    """
+    Executor for explode operator to explode nested list.
+
+    Arguments:
+        node (AbstractPlan): ExplodePlan
+
+    """
+
+    def __init__(self, node: ExplodePlan):
+        super().__init__(node)
+        self._column_list = node.column_list
+
+    def validate(self):
+        pass
+
+    def exec(self) -> Iterator[Batch]:
+        for batch in self.children[0].exec():
+            if not batch.empty():
+                frames: pd.DataFrame = batch.frames
+
+                first_row = frames.iloc[0]
+                len_arr = []
+                for column in self._column_list:
+                    col_name = column.col_name
+                    if col_name not in frames.columns:
+                        raise KeyError("Column doesn't exist in the data frame: {}".format(column.col_name))
+                    element = first_row[col_name]
+                    if type(element) == list and len_arr == []:
+                        col_list = frames[col_name].values.tolist()
+                        len_arr = [len(r) for r in col_list]
```

Comment:
This logic seems a little weird to me. Should we get len_arr from all exploded columns and assert they are equal?
BTW: if the exploded column has to be a list, then we should raise exception if it is not. For example, is a query like `EXPLODE SELECT id, YOLO(frame).label FROM video, id;` allowed?